newIDE/app/src/ProjectsStorage/LocalProjectWriter.js

```diff
@@ -80,6 +86,72 @@ export default class LocalProjectWriter {
     }
   };
 
+  static saveProjectAs = (project: gdProject): Promise<void> => {
+    const fileSystem = assignIn(new gd.AbstractFileSystemJS(), localFileSystem);
+    let filepath = project.getProjectFile();
+    const projectPath = path.dirname(project.getProjectFile());
+    const browserWindow = electron.remote.getCurrentWindow();
+    const options = {
+      defaultPath: projectPath + '\\game.json',
```

Comment:
Prefer using path.join for cross platform separators:
`defaultPath: path.join(projectPath, "game.json")`

Import path like this:
`const path = optionalRequire('path');`